### PR TITLE
feat: interface improvements

### PR DESCRIPTION
**Pull request by Matiasnickolas**
This PR adds some improvements for user interface.

- [x] Change project icon
- [x] Change project title
- [x] Show alert on open port failures
- [x] Show messages to await pos response

image

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset="utf-8" />
-  <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+  <link rel="icon" href="%PUBLIC_URL%/icon.ico" />
```
Comment (mastudillot) [inline public/index.html:6]: Rename to favicon

**Review by mastudillot — COMMENTED** [review]
check my comments

File: public/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
```
Comment (mastudillot) [inline public/index.html:9]: use a spanish description

**Review by mastudillot — APPROVED** [review]
GJ 🤙🏼

**Review by DiegoHBustamante — APPROVED** [review]

**Pull request merged**